src/services/email.js

```diff
@@ -6,8 +6,13 @@ const {
 } = require('~/configs/config')
 const { createError } = require('~/utils/errorsHelper')
 const { TEMPLATE_NOT_FOUND } = require('~/consts/errors')
+const path = require('node:path')
```

Comment:
And move this import before our custom modules. 
```suggestion
const path = require('path')
```